app/Http/Resources/ItemResource.php

```diff
@@ -82,8 +82,11 @@ private function getAuthor()
             return $this['item']->author_name;
         }
 
-        return collect($this['webumenia_item']->authors)
-            ->map(fn(string $author) => formatName($author))
+        return collect($this['webumenia_item']->authorities)
+            ->map(
+                fn(object $authority) => formatName($authority->name) .
+                    ($authority->role ? ' - ' . $authority->role : '')
```

Comment:
```suggestion
                    (isset($authority->role) && $authority->role ? ' - ' . $authority->role : '')
```

to avoid `Undefined property: stdClass::$role` exception and make it possible to merge now - and not necessarily wait for webumenia api update.